sklearn/datasets/_samples_generator.py

```diff
@@ -702,7 +702,8 @@ def make_moons(n_samples=100, shuffle=True, noise=None, random_state=None):
 
 
 def make_blobs(n_samples=100, n_features=2, centers=None, cluster_std=1.0,
-               center_box=(-10.0, 10.0), shuffle=True, random_state=None):
+               center_box=(-10.0, 10.0), return_centers=False,
+               shuffle=True, random_state=None):
```

Comment:
We should introduce the new parameter at the end to avoid breaking existing code